src/components/Header/Header.jsx

```diff
@@ -3,10 +3,12 @@ import Logo from "./Logo/Logo.jsx";
 import styles from './Header.module.css';
 import BtnAdd from "./BtnAdd/BtnAdd.jsx";
 import Search from "./Search/Search.jsx";
-import PropTypes from "prop-types";
+import PropTypes from 'prop-types';
+import {useParams} from 'react-router-dom';
```

Comment:
I see that useParams is not used in this file

Reply:
Fixed